### ISSUE TITLE
Release plan - v18.x Maintenance LTS

### ISSUE THREAD

**Issue by BethGriggs** [nodejs/Release#737]
_Draft schedule - all dates subject to change_

Version | Release Date | Releaser
--------|--------------| ---------
v18.12.0 | 2022-10-25 | @ruyadorno and @RafaelGSS (LTS Transition)
v18.12.1 | 2022-11-04 | @juanarbol (Security)
v18.13.0 | 2023-01-03 | @danielleadams 
v18.14.0 | 2023-01-31 | @juanarbol 
v18.14.1 | 2023-02-16 | @juanarbol (Security)
v18.14.2 | 2023-02-21 | @MylesBorins 
v18.15.0 | 2023-03-07 | @juanarbol 
v18.16.0 | 2023-04-11 | @danielleadams 
v18.16.1 | 2023-06-20 | @RafaelGSS (Security)
v18.17.0 |  2023-08-09 | @danielleadams 
v18.18.0 | 2023-09-18 | @ruyadorno
v18.18.1 | 2023-10-10  | @richardlau 
v18.18.2 | 2023-10-13 | @RafaelGSS 
v18.19.0 |  2023-11-29  | @targos
v18.19.1 | 2023-02-14 | @RafaelGSS (Security)
v18.20.0 | 2024-03-26 | @richardlau 
v18.20.1 | 2024-04-03 | @RafaelGSS (Security)
v18.20.2 | 2024-04-10 | @RafaelGSS (Security)
v18.20.3 | 2024-05-21 | @richardlau 
v18.20.4 | 2024-07-08 | @RafaelGSS (Security)
v18.20.5 | 2024-11-12 | @aduh95 
v18.20.6 | 2025-01-21 | @RafaelGSS (Security)
v18.20.7 | 2025-02-20 | @aduh95 
v18.20.8 | 2025-03-27 | @richardlau 

<details>
<summary>Current</summary>

Version | Release Date | Releaser
--------|--------------| ---------
v18.0.0 | 2022-04-19 |  @BethGriggs 
v18.1.0 | 2022-05-03 |  @targos
v18.2.0 | 2022-05-11 |  @BethGriggs prepared by @RafaelGSS 
v18.3.0 | 2022-05-31 |  @bengl
v18.4.0 | 2022-06-16 |  @danielleadams 
v18.5.0 | 2022-07-07 |  @RafaelGSS  (security)
v18.6.0 | 2022-07-13 |  @targos 
v18.7.0 | 2022-07-26 |  @danielleadams 
v18.x.x | 2022-08-09 |  @danielleadams 
v18.8.0 | 2022-08-24 |  @ruyadorno 
v18.9.0 | 2022-09-06 |  @RafaelGSS
v18.9.1 | 2022-09-22 |  @RafaelGSS (security)
v18.10.0 | 2022-09-28 |  @RafaelGSS
v18.11.0 | 2022-10-11 |  @danielleadams 

</details>

**Comment by targos** [comment]
@juanarbol do you still plan to prepare a release for tomorrow? No worries if you can't, but I'd like to know so I take time to do it.

**Comment by BethGriggs** [comment]
@targos do you still have time to pick it up (maybe delayed until Weds/Thurs)? I reached out to @juanarbol by other means and it seems he is unavailable this week.

**Comment by targos** [comment]
I'll do it

**Comment by danielleadams** [comment]
I'm still preparing v18.x - aiming for Thursday (8/11) release date.

**Comment by danielleadams** [comment]
> I'm still preparing v18.x - aiming for Thursday (8/11) release date.

Preparing v18.x is still ongoing - right now, it seems I am blocked by backporting https://github.com/nodejs/node/pull/44056 (I may try myself) because the rest of the merge conflicts/test failures build on top of that. Depending on if I can get that done in the next ~2 days, I may just work with @ruyadorno to get the giant 18.x release out next week for 8/23.

**Comment by RafaelGSS** [comment]
@nodejs/releasers We have a release target of 2022-09-20. However, the Security Release (v18.10.0) will be issued on 2022-09-22. Would be ok to postpone the regular release to at least 2 days after the Security Release (24/09)? That would be better for me since I'm part of both releases.

**Comment by danielleadams** [comment]
@RafaelGSS I think we could push it back a week, or even skip this next minor release in favor of the security release. As long as commits from main get pulled in before it goes LTS, I think it's fine.

**Comment by danielleadams** [comment]
@RafaelGSS oh, I see. I think that's fine because the next target date is 04/10. I can also pick it up for 27/09 if you want with all the recent commits, and then we can see if we should push the 04/10 one out 1 week too.

**Comment by krk** [comment]
Hi! Will there be a release between now and the LTS version? Would the LTS release actually only contain a version number bump commit?

**Comment by w3nl** [comment]
Can you update the schedule, maybe also with planned releases like the 19 release schedule?
I think it's good to know when a release is scheduled, e.g. if you wait for a fix is deployed.

**Comment by targos** [comment]
We will probably discuss and update the schedule in our monthly meeting next week.

**Comment by w3nl** [comment]
> We will probably discuss and update the schedule in our monthly meeting next week.

I see new version already for node 19, new version of 18 was scheduled for yesterday, so I hope 18.13 will be released soon.

**Comment by danielleadams** [comment]
@w3nl That's on me. I didn't get around to releasing this week, but I will have time this upcoming few days to prepare next week's release.

**Comment by w3nl** [comment]
@danielleadams no release yet?

Still waiting on a fix I reported on 15 okt and, fixed on 18 okt and merged to main om 20 okt.

And still not available in the latest 18.x, so I hope the new 18.x will released soon with the fix for the coverage.
18.12.x had only some security fixed, not other bug fixed.

If you need some help, please let it know.

**Comment by krk** [comment]
Is it possible to include https://github.com/nodejs/node/pull/45659 in the 18.13 release?

**Comment by vsmteam** [comment]
More desired candidates for 18.13.0 that could be a DoS if calls to certificate verification alternate with signing.

https://github.com/nodejs/node/pull/45377
https://github.com/nodejs/node/pull/45495

**Comment by RafaelGSS** [comment]
There will not be a v18.x release in December. I've updated our schedule. The next release is planned for early January. It will include all the fixes mentioned.

**Comment by ruyadorno** [comment]
hi @juanarbol, just wanted to confirm if you're still available for the next v18 release on January 31st.

**Comment by juanarbol** [comment]
> 

Yes! I do

**Comment by chanix95** [comment]
@juanarbol is there any plans to update npm package to 9.4.2 in the next release? My question is related to the remediation of security vulnerability merged in https://github.com/npm/cli/pull/6115 (CVE-2022-25881)

Thanks a lot!

**Comment by chanix95** [comment]
Just in case anyone was interested in my last question, the fix was included in v18.14.2 (PR: https://github.com/nodejs/node/pull/46673 )

**Comment by juanarbol** [comment]
I want to offer apologies for the delay in the v18.15.0 release. I updated the release table with the new date (march 7, 2023) while my new keys landed.

**Comment by name-yh2022** [comment]
Hello @targos,
Do you have plans to upgrade OpenSSL to version 3.0.12 in v18.19.0? I am concerned about whether the following vulnerability will be resolved. https://security.snyk.io/vuln/SNYK-ALPINE317-OPENSSL-6032385

**Comment by targos** [comment]
Yes, it will be part of it.

**Comment by kumarrishav** [comment]
Can we please land https://github.com/nodejs/node/pull/50186 on the upcoming v18?

This is a bug fix. Because of this we are unable to move to node 18 and have to use node 16.

**Comment by gabrielschulhof** [comment]
The Node-API team also needs https://github.com/nodejs/node/pull/50991 in v18.x in order to unblock https://github.com/nodejs/node-addon-api/pull/1409. Could you please add it to the upcoming v18 release?

**Comment by RafaelGSS** [comment]
Hi Folks,

Node.js v18 is in maintenance, implying we no longer perform scheduled releases to this release line. However, a release would only happen if one of these criteria is met:

* A nasty bug is encountered
* Security release
* One of @nodejs/releasers decides/wants to perform a release

**Comment by kumarrishav** [comment]
Now, that we have 18.x security release planned. can we land https://github.com/nodejs/node/pull/50186 on v18 now?

**Comment by gabrielschulhof** [comment]
Can we please add https://github.com/nodejs/node/pull/50991 to the next 18.x release? It's blocking https://github.com/nodejs/node-addon-api/pull/1409.

**Comment by gabrielschulhof** [comment]
May we please have https://github.com/nodejs/node/pull/51801 in the next release? It will help us with our next Node-API release.

**Comment by nicolo-ribaudo** [comment]
If there will be another minor release of v18.x, it would be very nice to have https://github.com/nodejs/node/pull/52762 included. The patch applies cleanly.

**Comment by richardlau** [comment]
The Node.js 18 release won't happen today -- I'll have to delay it as getting the test CI back into a working state takes priority.

**Comment by MikeMcC399** [comment]
Can anybody confirm that Node.js 18 EOL is still planned for today April 30, 2025?
I've been asked if the delay in releasing Node.js 24 affects the Node.js 18 EOL. I assumed no EOL delay, since the website is showing today's date, but I'd be grateful for confirmation.

![Image](https://github.com/user-attachments/assets/a65c2528-b504-48f3-a600-1d2ada0b7650)

**Comment by richardlau** [comment]
There are no plans to change the End-of-Life date of Node.js 18.
